src/app/app.component.ts

```diff
@@ -1,10 +1,10 @@
-import {Component} from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```

Comment:
@Chryce This component does not seem to implement the `OnInit` interface. If the import is no longer necessary, we should remove it.

Reply:
Thanks, @Victoradukwu, I have worked on this.